src/Core/System/Snippet/Command/InstallTranslationCommand.php

```diff
@@ -40,10 +43,27 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $locales = $this->getLocales($input);
-        $progressBar = $this->createProgressBar($output, \count($locales));
+        $metadata = $this->metadataLoader->getUpdatedMetadata($locales);
+
+        $localesRequiringUpdate = $metadata->getLocalesRequiringUpdate();
+        if ($localesRequiringUpdate === []) {
+            $output->writeln('All translations are up to date.');
+
+            return self::SUCCESS;
+        }
+
+        $localesDiff = array_diff($locales, $localesRequiringUpdate);
+        if ($localesDiff !== []) {
+            $output->writeln(\sprintf(
+                'The following locales are already up to date and will be skipped: %s',
```

Comment:
Does it make sense to output locales that are not updated? I would expect those that will be updated.

Reply:
It outputs locales that are up-to-date and will be skipped. `array_diff` returns the locales that are in `$locales` included (the local metadata) but not in `$localesRequiringUpdate`.